### ISSUE TITLE
Pretrained weights

### ISSUE THREAD

**Issue by AWbosman** [nvedant07/Fairness-Through-Robustness#2]
Dear authors, 

I have some networks that are pretrained and would like to retrain with your method. 
In your read.me you state this would be added soon. 
Do you have any tips or examples on how to use your repository?

Kindly, 
Annelot

**Comment by nvedant07** [comment]
Hi @AWbosman ,

Thanks for checking out our repo! It's been a while since I worked on this, but I believe what you're looking for is the regularized loss that reduced robustness bias by adding regularization during training.

I added this file under [code/regularized_loss.py](https://github.com/nvedant07/Fairness-Through-Robustness/blob/master/code/regularized_loss.py). This contains a class called ``RegularizedLoss`` which you can use in your training loop just like you use nn.CrossEntropyLoss in PyTorch. 

Let me know if you run into any issues!
